doc/install-on-docker.md

```diff
@@ -1,10 +1,53 @@
 # Installation on Docker
 
-You can install Ray on any platform that runs Docker. We do not presently publish Docker images for Ray, but you can build them yourself using the Ray distribution. Using Docker can provide a reliable way to get up and running quickly.
+You can install Ray on any platform that runs Docker. We do not presently publish Docker images for Ray, but you can build them yourself using the Ray distribution.
+
+Using Docker can streamline the build process reliable way to get up and running quickly.
 
 ## Install Docker
 
-The Docker Platform release is available for Mac, Windows, and Linux platforms. Please download the appropriate version from the [Docker website](https://www.docker.com/products/overview#/install_the_platform).
+### Mac, Linux, Windows platforms
+
+The Docker Platform release is available for Mac, Windows, and Linux platforms. Please download the appropriate version from the [Docker website](https://www.docker.com/products/overview#/install_the_platform) and follow the corresponding installation instructions.
```

Comment:
Yeah, the official Linux instructions are not very relevant to our use case. I'll add a link to the alternate instructions.